src/main/java/io/dropwizard/sharding/sharding/ShardManager.java

```diff
@@ -32,7 +32,7 @@
 @ToString
 @Slf4j
 public class ShardManager {
-    public static final int MIN_BUCKET = 0;
+    private static final int MIN_BUCKET = 0;
```

Comment:
Why this difference? one public, one private?

Reply:
This is IntelliJ's doing. Auto corrected the data types. MAX_BUCKET must have been used elsewhere, while MIN_BUCKET might be restricted to the current scope. 

Fixed it.